dockerfiles/Dockerfile.prod

```diff
@@ -1,6 +1,6 @@
 # node-build stage
```

Comment:
I removed all the openshift references, since we will be switching to doing builds on Github and not from Openshift any more.

So this mean we have some workflow change to existing process on Openshift Deployment. I am guessing should be minor and the process has been established from other TL project in Openshift

Reply:
I think with this process the Dockerfile.openshift/Dockerfile.prod can just be deleted. That was only if this needed to build on Openshift and we'll just switch that to pull from GH

Reply:
@jonespm , I don't think we can delete `openshift/Dockerfile.prod` since 

Dockerfile: CMD ["/usr/bin/supervisord", "-c", "/code/deploy/supervisor_docker.conf"]
Dockerfile.Prod: CMD ["/code/start_backend.sh"]

`RUN npm run build:frontend`,  This command is missing from `Dockerfile` I don't see JS Bundle is made with Dockerfile . Here is the output from that build https://github.com/pushyamig/canvas-app-explorer/actions/runs/11746094247/job/32725028305

Reply:
@jonespm Let me know if you have any further concerns on this change. I will work to resolve it.

Reply:
I see what you mean. It would be fine running it through supervisord in prod too (possibly better) but we don't need to run both the frontend and backend processes since the frontend is static. 

So we'd need a to set it up so it only starts the backend but starts the frontend on development. 

I think the way this would be configured would be to just have the backend configured with `autostart=true` and the frontend be `autostart=false`. Then either through docker-compose or the startup script also start the frontend.

Reply:
So add environmental variable  in docker compose and allow it to run FE and BE during dev not in the prod